### ISSUE TITLE
Synchronizing  ONE way ONLY

### ISSUE THREAD

**Issue by hitam4450** [bitfireAT/davdroid#531]
I use davdroid 0.8 on my ANDROID Tab with Lollipop 5.0.2, established a sync account successfully. However, the data only will be synced one way form OC 8.0.3 calendar to the tablet. However, synchronizing the other way around FROM tablet to OC 8.0.3 calendar does not work at all!

Why ? What went wrong?
regards, hiatm4450

**Comment by phy25** [comment]
Can you provide log entries so that they can identify the problem? The logs can be viewed in this way: https://github.com/bitfireAT/davdroid/wiki/How-to-view-the-logs

**Comment by devvv4ever** [comment]
do you get any error messages or error notifications? if you're entering new events on the tablet did you make sure to choose the correct account from the dropdown?

**Comment by hitam4450** [comment]
@devvv4ever  No; i did  not get any error messages or notifications. Nothing at all!

**Comment by devvv4ever** [comment]
are you sure you entered the events on the tablet under the correct davdroid account? are you sure the added calendar is not read-only (you should see this in the davdroid setup where the resources are displayed)?

as stated above: logs from your device woulld be very (!) helpful. otherwise we can not find out what's the cause for this. most likely it is a permission problem on the OC service or some kind of mis-configuration... can you check the logs on your webserver after a forced sync from the device?

**Comment by hitam4450** [comment]
I investigated the situation.
After turning the SYNC feature on onto the ANDROID tablet, I established a test event in OC 8.03 it got synced onto the tablet. The other way around, adding an event onto ANDROID tablet got synced with the OC8.0.3. However, established events did not get synced.  However, moving the event to the established SYNC-CALENDAR works perfect by syncing BOTH WAYS!

As it holds true for many incidents like this: The biggest problem is located in front of the screen!  lol  :-)

Thanks & regards, Hitam4450
